codegenerator/cli/templates/dynamic/codegen/src/RegisterHandlers.res.hbs

```diff
@@ -137,22 +137,13 @@ let registerContractHandlers = (
             module(HyperFuelWorker.Make({
               let chain = chain
               let endpointUrl = "{{hyperfuel_config.endpoint_url}}"
-              let eventModLookup =
-                contracts
-                ->Belt.Array.flatMap(contract => contract.events)
-                ->EventModLookup.fromArrayOrThrow(~chain)
```

Comment:
Moved to the worker side and stopped using eventMod for Fuel worker.

The idea is to remove it for Evm as well.